src/documentation/pages/components-wip/footer/examples/_internal-brightspeed.vue

```diff
@@ -0,0 +1,121 @@
+<template lang="pug">
+  <ComponentExample title="Internal" id="internal-brightspeed" :tabs="exampleTabs">
```

Comment:
It does not look like the same footer.

![image](https://user-images.githubusercontent.com/86958774/176478516-801df605-4482-4770-a374-e0676aac46f2.png)